src/cmd/main.go

```diff
@@ -54,6 +54,9 @@ func NewAzTuiState() *AzTuiState {
 		a.App.SetFocus(subscriptionList.List)
 	})
 
+	tl := resourceviews.NewTUILogView(a.AppLayout)
+	a.AppLayout.AppendPrimitiveView(subscriptionList.List, true, 1)
+	a.AppLayout.AppendPrimitiveView(tl.View, false, 3)	
```

Comment:
This will shift indexes for going back functionality, could we add this view to the Grid directly instead of layout?